lib/core/src/Cardano/Wallet/DB.hs

```diff
@@ -146,3 +147,7 @@ newtype ErrWalletAlreadyExists
 -- (like for instance, the last known network tip).
 newtype PrimaryKey key = PrimaryKey key
     deriving (Eq, Ord)
+
+-- | Clean a database by removing all wallets.
+cleanDB :: Monad m => DBLayer m s t -> m ()
+cleanDB db = listWallets db >>= mapM_ (runExceptT . removeWallet db)
```

Comment:
I am not sure we do want to keep this in the source code. This is purely for testing.